### PR TITLE
Revised documentation for feature - Effectivity Satellite

### PR DESCRIPTION
**Pull request by DVNorbertAcatrinei**
Revised documentation for feature - Effectivity Satellite

### CODE REVIEW

File: docs/metadata.md [hunk 1]
```diff
@@ -825,14 +825,14 @@ Hashdiff aliasing allows you to set an alias for the `HASHDIFF` column.
     ```yaml
     eff_sat_customer_nation:
       vars:
-        source_model: 'v_stg_transactions'
-        src_pk: 'TRANSACTION_PK'
-        src_dfk: 'CUSTOMER_PK'
-        src_sfk: 'NATION_PK'
+        source_model: 'v_stg_order_customer'
+        src_pk: 'CUSTOMER_ORDER_PK'
+        src_dfk: '["ORDER_PK"]'
```
Comment (DVAlexHiggs) [inline docs/metadata.md:830]: This should be:

        src_dfk: 
             - 'ORDER_PK'

Comment (DVAlexHiggs) [reply docs/metadata.md:830]: Kind of  a moot point though because we're removing this syntax suggestion in the next release

Comment (DVNorbertAcatrinei) [reply docs/metadata.md]: Resolved. It's now:
```
     src_dfk: 
         - 'ORDER_PK'
```

**Review by DVAlexHiggs — APPROVED** [review]
Thank you

**Pull request merged**